### DISCUSSION TITLE
pytorch lightening support

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/MONAI#4831] [Q&A]
Hi,
Can we integrate MONAI Core components into pytorch lightening workflow? For example, can I still use transforms and networks implemented in MONAI Core if I want to do distributed training by lightening?

**Comment by Nic-Ma** [comment]
Hi @hw-ju ,

I think you can refer to the tutorial:
https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d_lightning.ipynb

Thanks.

**Reply by hw-ju** [reply]
Thanks for pointing to the tutorial @Nic-Ma!
